bundles/storage/org.eclipse.smarthome.storage.temporary/OSGI-INF/org.eclipse.smarthome.storage.temporary.TemporaryStorageService.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
It does and is tracked in other bundles, what do you recommend we do?

Reply:
It is not tracked but ignored in ESH and in newer bundles. `git rm` the xml files, add a `.gitignore` file containing `*.xml` to OSGI-INF.

Reply:
Oh, that reminds me: When starting liteserver, `ManualDiscoveryResource` throws errors about already being registered. That’s exactly the kind of error this would produce. Have you checked in an OSGi service XML file despite them explicitly being ignored in that directory? If so, you should remove it.

Reply:
For what it's worth: I saw that duplicate error today, too, in a rather virgin dev environment (based on `testing`).

```

org.osgi.service.component.ComponentException: The component name 'org.eclipse.smarthome.binding.hue.internal.discovery.ManualDiscoveryResource' has already been registered by Bundle 84 (org.eclipse.smarthome.binding.hue) as Component of Class org.eclipse.smarthome.binding.hue.internal.discovery.ManualDiscoveryResource
	at ...
```

Reply:
Jeah, i did in https://github.com/AlJungOrg/smarthome/pull/29/files, will remove it